contracts/UbiquityGovernance.sol

```diff
@@ -0,0 +1,204 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.3;
+
+import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Burnable.sol";
+import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Pausable.sol";
+import "./interfaces/IUbiquityGovernance.sol";
+import "./UbiquityAlgorithmicDollarManager.sol";
+
+contract UbiquityGovernance is
+    IUbiquityGovernance,
+    ERC20Burnable,
+    ERC20Pausable
+{
+    UbiquityAlgorithmicDollarManager public manager;
+
+    // solhint-disable-next-line var-name-mixedcase
+    bytes32 public DOMAIN_SEPARATOR;
+    // keccak256("Permit(address owner,address spender,
+    //                   uint256 value,uint256 nonce,uint256 deadline)");
+    bytes32 public constant PERMIT_TYPEHASH =
```

Comment:
we should move that to the manager as it used both by ugov and uad

Reply:
we can also make one preset like OZ does

make one ERC20Ubiquity preset with one interface IERC20Ubiquity (almost equal to actual IUbiquityAlgorithmicDollar.sol)

Reply:
Actually there are so close that creating a preset make sense

Reply:
done ERCUbiquity.sol